components/about.tsx

```diff
@@ -1,81 +1,101 @@
+              changes, both in the short-term and long-term.
             </p>
-            <div className="mx-auto max-w-xs sm:flex sm:max-w-none sm:justify-center sm:gap-4">
-              <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-primary-500 text-white-main hover:bg-primary-600 sm:mb-0 sm:w-auto"
-                  href="https://www.youtube.com/nang88"
-                  target="_blank"
-                >
-                  Watch Nang
-                </a>
-              </div>
-              <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-secondary-main text-white-main hover:bg-gray-800 sm:mb-0 sm:w-auto"
-                  href="https://www.youtube.com/@PansDay"
-                  target="_blank"
-                >
-                  Watch Pan
-                </a>
-              </div>
+            <h3 className="mb-2 text-xl font-semibold">
+              Current features include:
+            </h3>
+            <ul className="mb-8 list-inside list-disc text-gray-600">
+              <li>Codebase context: talk to your code</li>
+              <li>Built-in PearAI chat</li>
+              <li>Inline AI prompting and diff changes</li>
+              <li>AI debugging, including errors shown in terminal</li>
+              <li>And many more!</li>
+            </ul>
+            <h2 className="mb-4 text-2xl font-bold">Founders</h2>
+            <div className="space-y-4 text-gray-600">
+              <p>
+                PearAI is founded by Pan and Nang, both former software
+                engineers who worked on developer tooling at companies like
+                Meta, Coinbase, and high-frequency trading firms.
+              </p>
+              <p>
+                Passionate about empowering individuals, they recognize code and
+                media as the most scalable tools a single person can utilize.
+                With a combined following of over 500k subscribers and over 30
+                million views, they&apos;ve decided to combine media and code,
+                bringing the developer community along for the ride.
+                They&apos;re building the next open-source AI-powered code
+                editor, PearAI, with full transparency through videos,
+                livestreams, and direct Discord calls/chats.
+              </p>
             </div>
           </div>
+
+          {/* Right column */}
+          <div className="lg:w-1/2">
+            <VideoCard
+              src="https://www.youtube.com/embed/ZCzz3UXDazY"
+              title="FryingPan's Latest YouTube video"
+              description="FryingPan discusses the evolution from a creator-brand connection idea to PearAI, an open-source AI-powered code editor. He highlights how this tool aims to revolutionize developer efficiency."
+            />
```

Comment:
```diff
            </ul>
+           <div className="lg:hidden">
+             <VideoCard
+               src="https://www.youtube.com/embed/ZCzz3UXDazY"
+               title="FryingPan's Latest YouTube video"
+               description="FryingPan discusses the evolution from a creator-brand connection idea to PearAI, an open-source AI-powered code editor. He highlights how this tool aims to revolutionize developer efficiency."
+             />
+           </div>            
            <h2 className="mb-4 text-2xl font-bold">Founders</h2>
            <div className="space-y-4 text-gray-600">
              <p>
                PearAI is founded by Pan and Nang, both former software
                engineers who worked on developer tooling at companies like
                Meta, Coinbase, and high-frequency trading firms.
              </p>
              <p>
                Passionate about empowering individuals, they recognize code and
                media as the most scalable tools a single person can utilize.
                With a combined following of over 500k subscribers and over 30
                million views, they&apos;ve decided to combine media and code,
                bringing the developer community along for the ride.
                They&apos;re building the next open-source AI-powered code
                editor, PearAI, with full transparency through videos,
                livestreams, and direct Discord calls/chats.
              </p>
            </div>
          </div>

          {/* Right column */}
          <div className="lg:w-1/2">
+           <div className="hidden lg:block">
              <VideoCard
                src="https://www.youtube.com/embed/ZCzz3UXDazY"
                title="FryingPan's Latest YouTube video"
                description="FryingPan discusses the evolution from a creator-brand connection idea to PearAI, an open-source AI-powered code editor. He highlights how this tool aims to revolutionize developer efficiency."
              />
+           </div>
```

This shows the first video in between the two texts (Pan's suggestion) on mobile. Desktop display is unaffected. 
![image](https://github.com/trypear/pear-landing-page/assets/87584757/b1eaafa2-42ab-40f7-bfc8-5020a6da6d38)

![image](https://github.com/trypear/pear-landing-page/assets/87584757/aa372e74-66d6-4c80-9602-9fddfa00cf0b)
![image](https://github.com/trypear/pear-landing-page/assets/87584757/5961e4ac-64d9-4348-8e4e-209c0f535451)